### PR TITLE
Added get & deserialize to metrics in TensorFlow frontend

### LINKED ISSUE #9680: deserialize

### LINKED ISSUE #9681: get

### PR DESCRIPTION
**Pull request by AdvancedMicroDev**
closes #9680 
closes #9681

### CODE REVIEW

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @AdvancedMicroDev 
Thanks for contributing to ivy😊 and for doing such great work👏

Just one single change, could you move these functions to the `activations` submodule?
After that, it will be good to be merged🙂

Thanks and happy coding✌

**Comment by AdvancedMicroDev** [comment]
> Hey @AdvancedMicroDev Thanks for contributing to ivy😊 and for doing such great work👏
> 
> Just one single change, could you move these functions to the `activations` submodule? After that, it will be good to be merged🙂
> 
> Thanks and happy coding✌

Hi @hirwa-nshuti `get` & `deserialize` already exist for `activations` submodule. I have just modified them for the `metrics` submodule. So, is it correct here? Thanks in advance!

File: ivy/functional/frontends/tensorflow/keras/metrics.py [hunk 1]
```diff
@@ -292,3 +292,36 @@ def cosine_similarity(y_true, y_pred):
...
+    "jax": (
+        "float16",
+        "float32",
+        "float64",
+    ),
+}
+
+
+# get 
+def get(name, custom_objects=None):
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/tensorflow/keras/metrics.py:326]: From [Tensorflow docs](https://www.tensorflow.org/api_docs/python/tf/keras/metrics/get) this function doesn't have `custom_objects` argument

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_metrics.py [hunk 2]
```diff
@@ -860,3 +918,85 @@ def test_tensorflow_cosine_similarity(
...
+    simple_test_two_function(
+        fn_name=fn_name,
+        x=data[0],
+        frontend="tensorflow",
+        fn_str="get",
+        dtype_data=dtype_data[0],
+        rtol_=1e-01,
+        atol_=1e-01,
+        ivy_submodules=["keras", "metrics"],
+        framework_submodules=["keras", "activations"],
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_metrics.py:1001]: framework submodules should be `["keras", "metric"]`

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Check the needed changes below.

File: ivy/functional/frontends/tensorflow/keras/metrics.py (hunk 1 above)
Comment (AdvancedMicroDev) [reply ivy/functional/frontends/tensorflow/keras/metrics.py:326]: Yes, correct. I have made the required change locally & it works 👍

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_metrics.py [hunk 3]
```diff
@@ -2,12 +2,70 @@
...
+import tensorflow as tf
+
 
 # local
 import ivy_tests.test_ivy.helpers as helpers
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# Helper function for get & deserialize.
+def simple_test_two_function(
```
[the remark this answers is not in the document]
Comment (AdvancedMicroDev) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_metrics.py:15]: Did this too 👍

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_metrics.py (hunk 2 above)
Comment (AdvancedMicroDev) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_metrics.py:1001]: if I change the `activations` to `metrics`, it results in all tests being failed...

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
The tests still need to be targeted to `metrics` submodule
